ui/src/app/pages/access/data-access-requirements.spec.tsx

```diff
@@ -1902,4 +1906,105 @@ describe('DataAccessRequirements', () => {
       );
     });
   });
+
+  describe('CompletionBanner credits display', () => {
+    beforeEach(async () => {
+      registerApiClient(InstitutionApi, new InstitutionApiStub());
+      registerApiClient(ProfileApi, new ProfileApiStub());
+
+      profileStore.set({
+        profile: {
+          ...ProfileStubVariables.PROFILE_STUB,
+          accessModules: {
+            modules: initialRequiredModules.map((module) => ({
+              moduleName: module,
+              completionEpochMillis: 1,
+            })),
+          },
+          initialCreditsLimit: 300,
+          initialCreditsUsage: 50,
+          initialCreditsExpirationEpochMillis: oneYearFromNow(),
+        },
+        load,
+        reload,
+        updateCache,
+      });
+
+      serverConfigStore.set({
+        config: {
+          ...defaultServerConfig,
+          enableInitialCreditsExpiration: true,
+        },
+      });
+    });
+
+    it('should show remaining credits when initial credits feature is enabled', () => {
```

Comment:
not sure what you mean by "initial credits feature is enabled" here

Reply:
Oh, these are old tests that must be showing up as changes. 

The three tests that you mention here are from the interim of when initial credit expiration was being introduced. Not all environments had this enabled and users had not yet been backfilled. This is no longer the case. These tests should be removed, but I suspect that the `enableInitialCreditsExpiration` feature flag should be removed. I will remove the tests and handle the feature flag in a separate PR.